README.md

```diff
@@ -12,12 +12,19 @@ and checks code against the coding standards used in CakePHP.
 PSR2 standard. If you want to check against the historical CakePHP coding
 standard use any of the `1.x` releases.
 
+## Which version should I use?
+| CakePHP  | Sniffer |
+| -------- | ------- |
+| 2.x  | 1.0.4  |
```

Comment:
^1.0.4 IMO

Reply:
I wasn't sure if it should be a tag or a composer constraint. I went with a tag

Reply:
Tag is not too helpful. Either `>=` or semver `^` or something doesnt look it down semantically.